modules/js/planetaryFinal.js

```diff
@@ -76,129 +101,114 @@ function addPlanetaryGearComposite(centerX, centerY, constraintX, constraintY){
-  changeBodyPlanetary(0);
-}
-// see smallGear1()
-function largeGear2(){
-  Body.setAngle(compositeArray[0].bodies[0], 0)
-  Body.setAngle(compositeArray[1].bodies[0], 0)
-  radius = 80;
+
+function gear2(rad) {
+  Body.setAngle(compositeArray[0].bodies[0], 0);
+  Body.setAngle(compositeArray[1].bodies[0], 0);
+  radius = rad;
   compositeArray[0].radius = radius;
-  steps = (0.25 * radius)*2;
-  toothWidthDegree = 2;
-  toothWidth = (toothWidthDegree/conversionFactor);
+  steps = 0.25 * radius * 2;
+  toothWidthDegree = 4;
+  toothWidth = toothWidthDegree / conversionFactor;
   changeBodyPlanetary(0);
 }
 ////////////////////////////////////////////////////////////////////////////////////////////////
-Events.on(engine, 'beforeUpdate', function(event) {
-  Body.setPosition(compositeArray[1].bodies[1],compositeArray[1].bodies[0].position)
-  for(var i = 0; i<compositeArray.length; i++){
-    if(compositeArray[i].constraints[0]){
+Events.on(engine, "beforeUpdate", function (event) {
+  Body.setPosition(
+    compositeArray[1].bodies[1],
+    compositeArray[1].bodies[0].position
+  );
+  for (let i = 0; i < compositeArray.length; i++) {
+    if (compositeArray[i].constraints[0]) {
       compositeArray[i].constraints[0].render.visible = true;
     }
   }
   // track brace distance
-  var x1 = compositeArray[1].bodies[0].position.x
-  var x2 = compositeArray[0].constraints[0].pointA.x
-  var y1 = compositeArray[1].bodies[0].position.y
-  var y2 = compositeArray[0].constraints[0].pointA.y
-  planetaryBrace = Math.floor(Math.sqrt( (x1-x2)*(x1-x2) + (y1-y2)*(y1-y2) ))
-})
+  const x1 = compositeArray[1].bodies[0].position.x;
+  const x2 = compositeArray[0].constraints[0].pointA.x;
+  const y1 = compositeArray[1].bodies[0].position.y;
+  const y2 = compositeArray[0].constraints[0].pointA.y;
+  planetaryBrace = Math.floor(
+    Math.sqrt((x1 - x2) * (x1 - x2) + (y1 - y2) * (y1 - y2))
+  );
+});
 ////////////////////// RUN /////////////////////////////
 // add initial parts first time cod runs
-addGearComposite((window.innerWidth)*(0.75*0.45)+(radius+(toothHeight*0.6)), (window.innerHeight)*(0.65));
-addPlanetaryGearComposite((window.innerWidth)*(0.75*0.45)-(radius+(toothHeight*0.6)), (window.innerHeight)*(0.65),(window.innerWidth)*(0.75*0.45)+(radius+(toothHeight*0.6)),(window.innerHeight)*(0.65));
+addGearComposite(
+  window.innerWidth * (0.75 * 0.45) + (radius + toothHeight * 0.6),
+  window.innerHeight * 0.65
+);
+addPlanetaryGearComposite(
+  window.innerWidth * (0.75 * 0.45) - (radius + toothHeight * 0.6),
+  window.innerHeight * 0.65,
+  window.innerWidth * (0.75 * 0.45) + (radius + toothHeight * 0.6),
+  window.innerHeight * 0.65
+);
 compositeArray[1].isMotor = true;
 compositeArray[0].lock = true;
 compositeArray[0].motorSpeed = 0.051;
 compositeArray[1].motorSpeed = 0.051;
 compositeArray[1].motorDir = -1;
 compositeArray[0].motorDir = 1;
-console.log(compositeArray[0].constraints[0])
+console.log(compositeArray[0].constraints[0]);
```

Comment:
Not sure how many console.log statements are needed to be kept around.  This seems useful for development but not production.